can/interfaces/pcan/pcan.py

```diff
@@ -2,15 +2,18 @@
 Enable basic CAN over a PCAN USB device.
 """
 
+from .basic import *  # Do this first since the wildcard imports a lot
```

Comment:
Why did you put this above the builtins?

Reply:
Ah, good point. I think it was due to a linter complaining. I'm not sure though ... it seemed clever at the time. ^^ Should I change it?

Reply:
Unless there's a good reason i would keep the usual order of imports: builtins, third party libraries and then the rest. But it is not a big deal. I'd rather remove the wildcard import altogether, but that's outside the scope of your PR. :smile:

Reply:
Ah, since the wildcard cause the issues in the first place, I actually removed it. To somewhat reduce the amount of imports, I moved three simple dicts/lists to the `basic.py`. Reviewing the commit in isolation is probably the easiest, the changes are really simple.